src/main/kotlin/io/snyk/plugin/ui/SnykBalloonNotifications.kt

```diff
@@ -92,6 +101,21 @@ class SnykBalloonNotifications {
             }
         )
 
+        fun showScanningReminder(project: Project) = showInfo(
+            "Scan your project for security vulnerabilities and code issues",
+            project,
+            NotificationAction.createSimpleExpiring("Run scan") {
+                project.service<SnykTaskQueueService>().scan()
+                project.service<SnykAnalyticsService>().logAnalysisIsTriggered(
```

Comment:
move logAnalysisIsTriggered() invocation into scan() ?

Reply:
we'll do it in separate PR.